client/opentracing/src/main/java/io/opentracing/APMSpanBuilder.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.opentracing;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.hawkular.apm.client.api.reporter.TraceReporter;
+
+/**
+ * This class is used to build the information used to create a Trace node.
+ *
+ * @author gbrown
+ */
+public class APMSpanBuilder extends AbstractSpanBuilder {
```

Comment:
Possibly, but they have implemented them as separate abstract classes, so may keep this for now.